### DISCUSSION TITLE
當 NASA 太空人的有趣條件

### DISCUSSION THREAD

**Discussion by 1chooo** [1101-GS4719-C/Python-Community-GS4719-C#1616] [閒聊八卦]
- 文章出處：https://www.brain.com.tw/news/articlecontent?ID=18201

首先這是一篇時代久遠的文章，因為在個版上滑到覺得很有趣，畢竟條件出自 NASA，因此想做個分享。

當太空人，這個夢想一直吸引著人們，但實現這個夢想並非易事。自從美國於1959年啟動太空計畫以來，NASA就一直在招募、培訓、並選擇出能夠飛向太空的太空人。然而，這個機會一直相當難得，並在半個多世紀中僅有20期太空人成功入選，第21期於近期開始。以下為原文 NASA 所要的太空人條件：

1. 有學士以及更高的工程、生物、物理、或數學學位。 
2. 在學位領域任專職工作三年以上，或駕駛噴射機1000小時以上。
3. 碩士學位可抵專職工作一年，博士學位可抵專職工作三年。
4. 教育工作者自動抵換專職工作三年。
5. 經矯正過的視力，包括雷射手術，必需為2.0
6. 坐姿血壓上限為收縮壓140、舒張壓90。
7. 身高在157到185公分之間

雖然擁有這些資格不代表就能入選，但它們是第一關。候選人必須經歷書面審核、面試、身體檢查，以確保他們在健康和能力上都符合要求。

一旦入選，準太空人會接受長達四年半到五年的基礎訓練，包括學科和術科的課程，以及軍事化的生存技能培訓。之後，他們將進行任務訓練，並接受資深太空人的指導，以準備面對太空任務的挑戰。

**以下是一些關於成為太空人的挑戰和看法：**
1. 難上加難：成為太空人的過程極其困難，需要充分的學術背景和專業經驗。每一個步驟都經過嚴格的篩選，只有極少數的人能夠通過，況且要能兼顧學術背景以及專業經驗，這兩件事情對我來說有點衝突，必堅都是固定的，人也需要休息。
2. 健康與體能：對於現在的人要能完成手邊的事又要抽空去做休閒運動，本就是一件非常難的事情，畢竟現在生活中的壓力來自四面八方，可能因為 A 事件耽誤了 B 事件，導致壓力越來越大，不過大家有空還是多點時間運動，健康好身體。
3. 夢想與冒險：要非常確定夢想就是當太空人，因為要能有一定的學術累積是需要花大量時間的，可能身邊的人做的事情也跟自己不盡相同，因此要非常確立自己的能想與麵包是否都能兼顧，讓身心靈都可以負荷。

總而言之當太空人的條件真的是非常嚴苛，不僅要具備嚴格的學術背景、專業經驗和優秀的健康狀況，而且條件也相較於一般的工作嚴苛太多，不見得每個有實力的人都具備這麼優良的體格，看完真的更敬佩那些能夠去太空展開太空之旅的偉大科學家們，同樣也尊重在各領域各自努力發光發熱的人們！

歡迎大家一起討論當太空人的條件又或是大家的夢想！

**Comment by spermwhale464** [comment]
https://technews.tw/2020/09/02/future-space-travel/
達不到標準沒關係，之後太空旅遊不是夢，只是一趟就要掏空你的荷包了
